packages/piro/test/Main_AnalysisDriver_Tpetra.cpp

```diff
@@ -77,10 +82,13 @@ int main(int argc, char *argv[]) {
   // Initialize MPI
   Teuchos::GlobalMPISession mpiSession(&argc,&argv);
   int Proc=mpiSession.getRank();
+
+  Teuchos::RCP<Teuchos::Comm<int> >  appComm;
+
 #ifdef HAVE_MPI
-  MPI_Comm appComm = MPI_COMM_WORLD;
+  appComm = Teuchos::rcp(new Teuchos::MpiComm<int>(Teuchos::opaqueWrapper(MPI_COMM_WORLD)));
```

Comment:
Why not include `Tpetra_Core.hpp` and write this:
```c++
auto appComm = Tpetra::getDefaultComm ();
```

Reply:
Thanks @mhoemmen, I'll do as you suggested.